layers/generate_layers.py

```diff
@@ -345,6 +355,118 @@ def makeLocalAuthorityDistricts():
         f.write(json.dumps(gj))
 
 
+# You have to manually download the GeoJSON file from https://geoportal.statistics.gov.uk/datasets/ons::output-areas-2021-boundaries-ew-bgc/explore and pass in the path here (until we can automate this)
+def makeCensusOutputAreas(raw_boundaries_path):
+    tmp = "tmp_census_output_areas"
+    os.makedirs(tmp, exist_ok=True)
+
+    # Build up a dictionary from OA code to properties we want
+    oa_to_data = {}
+
+    # Grab car availability data
+    run(
+        [
+            "wget",
+            "https://www.nomisweb.co.uk/output/census/2021/census2021-ts045.zip",
+            "-O",
+            f"{tmp}/census2021-ts045.zip",
+        ]
+    )
+    # Only get one file from the .zip
+    run(["unzip", f"{tmp}/census2021-ts045.zip", "census2021-ts045-oa.csv", "-d", tmp])
+    with open(f"{tmp}/census2021-ts045-oa.csv") as f:
+        for row in csv.DictReader(f):
+            oa_to_data[row["geography code"]] = summarizeCarAvailability(row)
+
+    # Grab population density
+    run(
+        [
+            "wget",
+            "https://www.nomisweb.co.uk/output/census/2021/census2021-ts006.zip",
+            "-O",
+            f"{tmp}/census2021-ts006.zip",
+        ]
+    )
+    run(["unzip", f"{tmp}/census2021-ts006.zip", "census2021-ts006-oa.csv", "-d", tmp])
+    with open(f"{tmp}/census2021-ts006-oa.csv") as f:
+        for row in csv.DictReader(f):
+            key = row["geography code"]
+            # The set of OAs in both datasets match. Let a KeyError happen if not.
+            oa_to_data[key]["pop_density"] = round(
```

Comment:
Oh yeah, the other simplification: https://www.nomisweb.co.uk/sources/census_2021_bulk has TS006, which directly gives us people per square km